app/src/containers/NewsDashboard/index.js

```diff
@@ -164,17 +167,17 @@ class NewsDashboard extends React.Component {
 
 	deleteNews = data => {
 		return new Promise((resolve, reject) => {
-			fetch(window.location.protocol + '//' + window.location.hostname
-				+ ConstantsList.PORT_IN_USE + "/api/v1/news/" + data._id,
-				{
-					method: "DELETE",
-					headers: {
-						'Accept': 'application/json',
-						'Content-Type': 'application/json',
-					},
-					credentials: 'include'
-				}
-			).then(res => {
+			getApiData({
+				version: 'v1',
+				endpoint: 'news' + data._id,
```

Comment:
Should there be a slash after news? Like this: endpoint: 'news/' + data._id, or is slash included in the data._id?

Reply:
Great find! You are right. I missed it. Fixing it.